libraries/api-cat-client/src/main/java/no/dcat/client/apicat/ApiCatClient.java

```diff
@@ -0,0 +1,39 @@
+package no.dcat.client.apicat;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ApiCatClient {
+    private String apiCatUrl;
+
+    public ApiCatClient(String apiCatUrl) {
+        this.apiCatUrl = apiCatUrl;
+    }
+
+    public OpenAPI parseApiSpec(String apiSpec) {
+        Map<String, String> body = new HashMap<>();
+        body.put("spec", apiSpec);
```

Comment:
Skulle ikke body være string, ikke json?

Reply:
jeg faktisk hadde det som body-string først, men da endret jeg det, på to grunn:
1) - at kanskje vi trenger ekstra instsruks parametrer, og da er det lurt å ha i post body ikke i querystring
2) like interface som parsebyurl